js/notify/electron-notify-preload.js

```diff
@@ -75,6 +78,22 @@ function setContents(event, notificationObj) {
 
     if (notificationObj.color) {
         container.style.backgroundColor = notificationObj.color;
+
+        if (notificationObj.color.match(/^(?:white|#fff(?:fff)?|rgba?\(\s*255\s*,\s*255\s*,\s*255\s*(?:,\s*1\s*)?\))$/i)) {
```

Comment:
Better to abstract this out. If in the future there is another lighted color (yellow, biege, etc.), we may have to use the black logo.